app/api/templates/specs/templateSchema.spec.js

```diff
@@ -167,5 +171,25 @@ describe('template schema', () => {
         expect(e).toBeInstanceOf(Ajv.ValidationError);
       }
     });
+
+    it('should be ok with other scenarios', async () => {
```

Comment:
Which scenario exactly? "Other scenarios" makes it really hard to know what it is this test is doing.

Reply:
Rafa, I don't agree that the test should describe specific implementations of the code, instead, it should describe functionality.
I agree that "other scenarios" may be a poor description, but the idea is a test that ensures that the validation is not going to throw an error when you are simply updating the template.